receiver/opencensusreceiver/testdata/config.yaml

```diff
@@ -1,19 +1,54 @@
 receivers:
+  # The following entry initializes the default OpenCensus receiver.
+  # The default values are specified here
+  #  https://github.com/open-telemetry/opentelemetry-service/blob/71589202609d7e787244076b631b45e219101867/receiver/opencensusreceiver/factory.go#L47-L56
+  # The full name of this receiver is `opencensus` and can be referenced in pipelines by 'opencensus'.
   opencensus:
+  # The following entry demonstrates configuring the common receiver settings:
+  # - endpoint
+  # - disable-backpressure
+  # For more information on the struct, refer to
+  # https://github:com/open-telemetry/opentelemetry-service/blob/71589202609d7e787244076b631b45e219101867/config/configmodels/configmodels.go#L142-L150
+  # This configuration has a name 'customname' with a full name of 'opencensus/customname'.
+  # To reference this configuration in a pipeline, use the full name `opencensus/customname`.
   opencensus/customname:
+    # The endpoint for this receiver will be: "0.0.0.0:9090".
     endpoint: 0.0.0.0:9090
+    # This receiver doesn't allow for back pressure.
+    disable-backpressure: true
+  # The following entry configures all of the keep alive settings. These settings are use during server initialization.
```

Comment:
>The following entry configures all of the keep alive settings.

This sounds like it is purely for configuring keep alive settings. It is a receiver which also shows how to configure keep alive settings.

>These settings are use during server initialization.
	
Unclear. What "server"?

Reply:
Server is referenced in several parts of the config and its documentation. I will update the description to say the settings are used to configure the underlying server used to connect to the endpoint.

Reply:
I do not think we should be using the word "server" when referring to the receiver endpoint. If we want to refer to the whole executable we should use the word "service", if it is about the specific receiver then let's just say "receiver".

Reply:
Using 'service' instead of 'server' is find with me. I will let other chime in since I have no strong feelings about it. However, if the final decision is to use 'service' instead of 'server' which is referenced in the config, sample yaml and comments across ocreceiver, I will change it in a follow up pr as to avoid making this pr any larger.

Reply:
Synced offline - This was a misunderstanding on my part and Tigran's proposal is valid. I will update the comment to use receiver/service and leave mention of server to the details of the keep alive parameters.